core/jobs/types/base_validation_errors_test.py

```diff
@@ -48,87 +55,98 @@ class BarModel(base_models.BaseModel):
 class FooError(base_validation_errors.BaseAuditError):
     """A simple test-only error."""
 
-    def __init__(self, model):
+    def __init__(self, model: Union[base_models.BaseModel, str]) -> None:
         super(FooError, self).__init__('foo', model)
 
 
 class BarError(base_validation_errors.BaseAuditError):
     """A simple test-only error."""
 
-    def __init__(self, model):
+    def __init__(self, model: Union[base_models.BaseModel, str]) -> None:
         super(BarError, self).__init__('bar', model)
 
 
 class AuditErrorsTestBase(core_test_utils.TestBase):
     """Base class for validator error tests."""
 
-    NOW = datetime.datetime.utcnow()
-    YEAR_AGO = NOW - datetime.timedelta(weeks=52)
-    YEAR_LATER = NOW + datetime.timedelta(weeks=52)
+    NOW: datetime.datetime = datetime.datetime.utcnow()
+    YEAR_AGO: datetime.datetime = NOW - datetime.timedelta(weeks=52)
+    YEAR_LATER: datetime.datetime = NOW + datetime.timedelta(weeks=52)
 
 
 class BaseAuditErrorTests(AuditErrorsTestBase):
 
-    def setUp(self):
+    def setUp(self) -> None:
         super(BaseAuditErrorTests, self).setUp()
         self.model = base_models.BaseModel(id='123')
 
-    def test_message(self):
+    def test_message(self) -> None:
         error = FooError(self.model)
 
         self.assertEqual(error.stderr, 'FooError in BaseModel(id="123"): foo')
 
-    def test_stdout(self):
+    def test_stdout(self) -> None:
         error = FooError(self.model)
 
         self.assertEqual(error.stdout, '')
 
-    def test_stderr(self):
+    def test_stderr(self) -> None:
         error = FooError(self.model)
 
         self.assertEqual(error.stderr, 'FooError in BaseModel(id="123"): foo')
 
-    def test_message_raises_type_error_if_assigned_a_non_string_value(self):
+    def test_message_raises_type_error_if_assigned_a_non_string_value(
+        self
+    ) -> None:
         class ErrorWithIntMessage(base_validation_errors.BaseAuditError):
             """Subclass that tries to assign an int value to self.stderr."""
 
-            def __init__(self, model):
-                super(ErrorWithIntMessage, self).__init__(123, model)
+            def __init__(
+                self, model: Union[base_models.BaseModel, str]
+            ) -> None:
+                # Here, str type is expected but for test purpose we're
+                # assigning it an int type. Thus to avoid MyPy error, we
+                # added an ignore here.
```

Comment:
Add a TODO here as this should be removed after we fully type the codebase.

Reply:
Added an existing issue #13059. To make consistency throughout the codebase.